Source/Core/DolphinWX/Main.cpp

```diff
@@ -135,8 +135,10 @@ bool DolphinApp::OnInit()
 
   VideoBackendBase::ActivateBackend(SConfig::GetInstance().m_strVideoBackend);
 
+  #ifdef USE_ANALYTICS
```

Comment:
I wouldn't mark the report methods for exclusion, mostly because you forgot at least two others; and there more coming in other PRs. If you _really_ want to exclude more than just sending it to the server, you should go after the contents of said methods; but I don't think thats necessary.